### PR TITLE
process glob matching in babel-cli (windows compat)

### PR DESCRIPTION
**Pull request by jden**
Unlike bash, Windows cmd shell does not expand glob patters (e.g. `*.js`) on its own - instead, commands have to implement that internally. You can simulate this in bash by using a string literal: `babel "*.js"`.

This patch uses the `glob` module (used by npm, et al) to add this capability to babel-cli, enabling better cross platform support.

Previously: 
```
> babel "*.js"
*.js doesn't exist
```

With patch:
```
> babel "*.js"
"use strict";

module.exports = require("babel-core");
"use strict";

module.exports = require("babel-core/polyfill");
"use strict";

module.exports = require("babel-core/register");
```

Tested and working on Windows 8 x64.

### CODE REVIEW

File: packages/babel-cli/bin/babel/index.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ var util      = require("babel-core").util;
 var each      = require("lodash/collection/each");
 var keys      = require("lodash/object/keys");
 var fs        = require("fs");
+var glob = require('glob');
```
Comment (sebmck) [inline packages/babel-cli/bin/babel/index.js]: Fix indentation and use double quotes please.

Comment (jden) [inline packages/babel-cli/bin/babel/index.js]: Thanks!

**Comment by jden** [comment]
The failing TravisCI tests have nothing to do with this pull request.

It would be nice if babel-ci were broken out into a separate repository for future development.

**Comment by sebmck** [comment]
Thanks!

> It would be nice if babel-ci were broken out into a separate repository for future development.

It's easier to maintain it in this repo since it's pretty rightly coupled and it's a part of the publish process and multi-repo releases are annoying to coordinate and maintain.

**Comment by sebmck** [comment]
Released as of 5.2.0, thanks!

**Pull request merged**